Gibe.DittoServices/Gibe.DittoServices.nuspec

```diff
@@ -1,21 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-  <metadata>
-    <id>Gibe.DittoServices</id>
-    <version>$version$</version>
-    <authors>Gibe Digital Ltd</authors>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>
-      Standard Gibe Services for Ditto
-    </description>
-    <dependencies>
-      <dependency id="Gibe.UmbracoWrappers" version="1.3.2.0" />
-      <dependency id="Our.Umbraco.Ditto" version="0.10.0" />
-      <dependency id="UmbracoCms.Core" version="7.3.7" />
-      <dependency id="System.ValueTuple" version="4.3.0" />
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="bin\Release\Gibe.DittoServices.dll" target="lib" />
-  </files>
+	<metadata>
+		<id>Gibe.DittoServices</id>
+		<version>$version$</version>
+		<authors>Gibe Digital Ltd</authors>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<description>
+			Standard Gibe Services for Ditto
+		</description>
+		<dependencies>
+			<dependency id="Gibe.UmbracoWrappers" version="1.3.2.0" />
+			<dependency id="Our.Umbraco.Ditto" version="0.10.0" />
+			<dependency id="UmbracoCms.Core" version="7.3.7" />
+			<dependency id="System.ValueTuple" version="4.3.0" />
+			<dependency id="Ninject" version="3.3.4" />
```

Comment:
Is this true? If we don't publish a dependency on Ninject, does the presence of the module cause problems, or only trying to access it? Do we know if anybody uses this without Ninject?

Reply:
I'm not sure, the "correct" config is to probably have a Gibe.DittoProcessors.Ninject package that provides the bindings.

I feel it's more correct to include dependencies that the package is using than not

Reply:
This is basically how we've done it for all our internal stuff, but I'm not sure if we've followed this pattern for our external stuff. I don't think just having ninject as an unused dependency would cause anybody any problems if they weren't using it, so if we think this is sufficient I'm happy to approve on those grounds.